### DISCUSSION TITLE
Possible to merge branches/PRs on a schedule?

### DISCUSSION THREAD

**Discussion by frederickfogerty** [renovatebot/renovate#9504] [Request Help]
Hi there Renovate team! Thanks so much for the great bot and tool, it really helps us a lot 🙌

Right now there is the pretty great schedule feature, which will make sure that any branches/PRs are only _opened_ on a schedule, but we were if it's possible to instead open the PRs/branches whenever, but only _merge_ them on a specified schedule?

Thanks 🙏

**Comment by rarkins** [comment]
Would it work for you if we enforced that merging must also happen within the *same* schedule you have already defined (i.e. the one that's used for creating branches/PRs)? (it's not done today, as it seems you're aware)

**Reply by frederickfogerty** [reply]
I think that might work fine and might be the cleanest from a configuration perspective.

The only downside I can think of is that there could be so many changes in one week that there is not enough time for CI to build/test all the changes within the time window, especially if the builds have to rebase every time. This can probably be alleviated by using a longer schedule window though.

Overall I'd say this could be quite a nice quick fix for this!
